yamscripts.yml

```diff
@@ -21,11 +21,10 @@ scripts:
     webpack: webpack --config webpack.system-addon.config.js --display-optimization-bailout
     css: node-sass --source-map true --source-map-contents system-addon/content-src/styles -o system-addon/css
     html: rimraf system-addon/prerendered && webpack --config webpack.prerender.config.js && node ./bin/render-activity-stream-html.js
-    # Copy over all of the system-addon directory
-    copy: cpx "system-addon/**/{,.}*" $npm_package_config_mc_dir/browser/extensions/activity-stream
-    version: node ./bin/update-version.js $npm_package_config_mc_dir/browser/extensions/activity-stream
+    # Copy over the entire repo
+    copy: cpx "./**/{,.}*" $npm_package_config_mc_dir/browser/extensions/activity-stream
+    version: node ./bin/update-version.js $npm_package_config_mc_dir/browser/extensions/activity-stream/system-addon
     copyPingCentre: cpx "system-addon/ping-centre/PingCentre.jsm" $npm_package_config_mc_dir/browser/modules
-    rmextra: rimraf "$npm_package_config_mc_dir/browser/extensions/activity-stream/{{,test/unit/}{content-src,ping-centre},data/locales.json}"
```

Comment:
Some of the things being removed here are the intermediate "built" files, e.g., locales.json, that currently aren't checked in to a-s or m-c.